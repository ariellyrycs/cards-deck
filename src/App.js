import CardContainer from './components/CardContainer.js';
import {ContextProvider} from './context/store';
import './App.css';

function App() {
  return (
    <div className="App">
      <ContextProvider>
        <CardContainer />
      </ContextProvider>
    </div>
  );
}

export default App;
