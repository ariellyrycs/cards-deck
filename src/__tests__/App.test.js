import App from '../App';
import { shallow, configure} from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

configure({ adapter: new Adapter() });

describe('should render App', () => {
  test('should render cardContainer', () => {
    const tree = shallow(<App />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});