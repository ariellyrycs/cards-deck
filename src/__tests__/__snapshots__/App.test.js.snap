// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App should render cardContainer 1`] = `
<div
  className="App"
>
  <ContextProvider>
    <CardContainer />
  </ContextProvider>
</div>
`;
